heartbeat/ocf-shellfuncs.in

```diff
@@ -962,7 +962,11 @@ ocf_default_trace_dest() {
 	if [ -n "$OCF_RESOURCE_TYPE" -a \
 			-n "$OCF_RESOURCE_INSTANCE" -a -n "$__OCF_ACTION" ]; then
 		local ts=`date +%F.%T`
-		__OCF_TRC_DEST=$HA_VARLIB/trace_ra/${OCF_RESOURCE_TYPE}/${OCF_RESOURCE_INSTANCE}.${__OCF_ACTION}.$ts
+		if [ -n "$OCF_TRACE_DIR" ]; then
+				__OCF_TRC_DEST=${OCF_TRACE_DIR}/${OCF_RESOURCE_INSTANCE}.${__OCF_ACTION}.$ts
```

Comment:
I'm thinking if it's sensible to anyway keep the parent directory `${OCF_RESOURCE_TYPE}/` as a common convention regarding the hierarchy of the directories/files? For example if an user specifies a common `trace_dir` for multiple resources, it'd probably still be convenient if they could easily find/distinguish them by the names of the directories?

Reply:
It makes perfect sense. I was thinking about another argument `trace_dir_suffix` or something alike, but this would be overkill, to my mind. I will append the `${OCF_RESOURCE_TYPE}` to the dir. If the user wants to be super explicit about the dir, he can use the `OCF_TRACE_FILE` after all.